src/diffusers/pipelines/fabric/pipeline_fabric.py

```diff
@@ -0,0 +1,552 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import warnings
+from typing import List, Optional, Union
+
+import numpy as np
+import torch
+from packaging import version
+from PIL import Image
+from transformers import CLIPTextModel, CLIPTokenizer
+
+from ...configuration_utils import FrozenDict
+from ...image_processor import VaeImageProcessor
+from ...models import AutoencoderKL, UNet2DConditionModel
+from ...models.attention import BasicTransformerBlock
+from ...models.attention_processor import LoRAAttnProcessor
+from ...schedulers import EulerAncestralDiscreteScheduler
+from ...utils import (
+    deprecate,
+    logging,
+)
+from ..pipeline_utils import DiffusionPipeline
+from . import FabricPipelineOutput
+
+
+logger = logging.get_logger(__name__)  # pylint: disable=invalid-name
+
+
+class CrossAttnProcessor:
```

Comment:
Is it an entirely new attention processor? If so, I have the following suggestion:

```suggestion
class FabricCrossAttnProcessor:
```